openlibrary/plugins/openlibrary/home.py

```diff
@@ -47,105 +46,31 @@ def GET(self):
         except:
             loans = None
 
-        return render_template(
+        page = render_template(
             "home/index", stats=stats,
             blog_posts=blog_posts,
```

Comment:
unclear returncart_list is being used anywhere -- it's rendered from within the template.